docs/index.md

```diff
@@ -36,7 +36,7 @@ provider "twingate" {
 - `api_token` (String, Sensitive) The access key for API operations. You can retrieve this
 from the Twingate Admin Console ([documentation](https://docs.twingate.com/docs/api-overview)).
 Alternatively, this can be specified using the TWINGATE_API_TOKEN environment variable.
-- `http_max_retry` (Number) Specifies a retry limit for the http requests made. This default value is 10.
+- `http_max_retry` (Number) Specifies a retry limit for the http requests made. This setting is 10.
```

Comment:
Why did this change?   the correct working is the original ("The default value is")

Reply:
thank you for the comment, yeah really not sure at which point I've change it, but agree with your proposal.
Will change it to original one

Reply:
regenerated docs